src/patternfly/components/AboutModalBox/examples/index.js

```diff
@@ -6,7 +6,7 @@ import AboutModalBoxExample from './about-modal-box-example.hbs';
 
 export const Docs = docs;
 
-export default (props) => {
+export default props => {
```

Comment:
@jenny-s51 not sure where this change came from, but i dont think its part of this PR so you can remove this change?